example/src/App.tsx

```diff
@@ -1,153 +1,91 @@
-import React, { useState } from 'react';
+import { useMemo, useState } from 'react';
 
-import {
-  StyleSheet,
-  View,
-  Button,
-  Switch,
-  Text,
-  TextInput,
-} from 'react-native';
-import RampSdk, { RampEventTypes } from '@ramp-network/react-native-sdk';
-import type { TAllEvents } from 'src/types';
+import { StyleSheet, View, Button, Switch, Text } from 'react-native';
+import RampSdk, { WidgetEventTypes } from '@ramp-network/react-native-sdk';
 
-export default function App() {
-  const [url, setUrl] = useState('https://app.dev.ramp-network.org');
-  const [hostApiKey, setHostApiKey] = useState(
-    'ku2jt8hctex6kwjsaz6ypphp26tbjysomwuj2yfj'
-  );
-  const [enabledFlows, setEnabledFlows] = useState<Array<string>>([
-    'ONRAMP',
-    'OFFRAMP',
-  ]);
-  const [defaultFlow, setDefaultFlow] = useState('ONRAMP');
-  let callbackText = '';
+type ENV = 'DEV' | 'STAGING' | 'PROD';
 
-  const [callbackTextLabel, setCallbackTextLabel] = useState('');
+export default function App() {
+  const [env, setEnv] = useState<ENV | undefined>('DEV');
+  const ramp = useMemo(() => {
+    if (!env) {
+      return undefined;
+    }
 
-  let rampSdk: RampSdk = new RampSdk()
-    .on(RampEventTypes.WIDGET_CLOSE, (event) => {
-      widgetClose(event);
-      callbackText += JSON.stringify(event) + '\n\n';
-      console.log('CALLBACKTEXT: ' + callbackText);
-      setCallbackTextLabel(callbackText);
+    return new RampSdk({
+      url: 'https://app.demo.ramp.network',
+      hostAppName: 'Calcutta',
```

Comment:
From where this name comes? Can it be like before?

Reply:
fixed